docs/pages/bare/using-expo-cli.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Using Expo CLI
+description: Learn how and why you can use Expo CLI instead of @react-native-community/cli in any React Native project.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+To use the local Expo CLI in a project created with `npx react-native init`, you will first need to install the `expo` package. This package includes the Expo Modules API and Expo CLI. For a complete explanation of how to install this, see the ["Install Expo modules" guide](/bare/installing-expo-modules.md). For most apps, it's as simple as running the following command in your project directory:
+
+<Terminal cmd={['npx install-expo-modules@latest']} />
+
+## Compiling and running your app
+
+Once you have the `expo` package installed, you can use the `npx expo run:ios` and `npx expo run:android` commands as alternatives to `npx react-native run-ios` and `npx react-native run-android`. You can pass in the `--device` flag to select a device/simulator to build for — this also works for any connected iOS device.
```

Comment:
Maybe add info about how the Expo run commands (ios) add additional contextual logging and enable you to build onto a connected device. Unlike the community CLI, you can do this without global side-effects, or opening xcode to manually provision the build.